lib/fast_counter.ex

```diff
@@ -38,11 +38,25 @@ defmodule Instruments.FastCounter do
   @spec increment(iodata) :: :ok
   @spec increment(iodata, integer) :: :ok
   @spec increment(iodata, integer, Statix.options()) :: :ok
-  def increment(name, amount \\ 1, options \\ []) do
+  def increment(name, amount \\ 1, options \\ [])
+
+  def increment(name, amount, []) do
```

Comment:
Would this be a little cleaner by creating a helper 
```elixir
defp get_table_key(name, options)
``` 
instead of multiple `increment`s?

Reply:
Perhaps if I inline it, with the performance difference being only a handful of microseconds I was trying to avoid additional function calls.

Reply:
Want me to run an analysis with that change with it inlined?

Reply:
Sure! I didn't really think there would be performance implications but I guess we're measuring microseconds here :D